src/sql/workbench/electron-browser/modelComponents/table.component.ts

```diff
@@ -250,4 +285,8 @@ export default class TableComponent extends ComponentBase implements IComponent,
 	public set selectedRows(newValue: number[]) {
 		this.setPropertyFromUI<azdata.TableComponentProperties, number[]>((props, value) => props.selectedRows = value, newValue);
 	}
+
+	public get forceFitColumns() {
+		return this.getPropertyOrDefault<azdata.TableComponentProperties, ColumnSizingMode>((props) => props.forceFitColumns, ColumnSizingMode.Default);
```

Comment:
are we changing the default behavior to no longer force fit?  will this impact the behavior anywhere in the product that current expect force fit?  maybe backup\restore dialogs, or agent extension?  If so, we should either retain the previous default or update those places to specify desired behavior.

Reply:
Right - looks like it's backwards.   Originally meant to have Default be the existing default behavior but along the way I must have got the wires crossed.  I'll fix the enum and the code so that Default is true.

Reply:
changing the default is probably fine if that makes sense, as long as we go through and add the properties to other places where using this component.